modules/nw-configure-ingress-access-logging.adoc

```diff
@@ -75,6 +75,8 @@ spec:
 [NOTE]
 ====
 The `syslog` destination port must be UDP.
+
+The `syslog` destination address must be an IP address. It does not support DNS hostname.
```

Comment:
Might need clarity about what _It_ is.  Maybe:

```suggestion
The `syslog` destination address must be an IP address. IPv4 does not support DNS hostnames.
```

Reply:
Hi @mburke5678,

Thanks for looking into it.

The address field does not support the DNS names.
`        syslog:
           address: 1.2.3.4  ----> Only supports IP address
           port: 10514
 `

_IPv4 does not support DNS hostnames._ statement will make logically incorrect.